sopel/modules/shrug.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+import sopel.module
```

Comment:
Best to `from sopel.module import commands` here and then just use `@commands()` below.